sockeye/arguments.py

```diff
@@ -248,7 +285,10 @@ def add_model_parameters(params):
     model_params.add_argument('--attention-coverage-num-hidden',
                               type=int,
                               default=1,
-                              help="Number of hidden units for coverage vectors. Default: %(default)s")
+                              help="Number of hidden units for coverage vectors. Default: %(default)s.")
+    model_params.add_argument('--attention-mhdot-heads',
```

Comment:
because in theory you could have a self-attending transformer encoder with x heads, but an RNN decoder using multi-head attention with y heads.